### PR TITLE
Added module dojoy/rpc/Service

### PR DESCRIPTION
**Pull request by HeikoStudt**
PR for adding dojox/rpc/service
(PS: I made a hickup with the author email in the commit.)

### CODE REVIEW

**Comment by HeikoStudt** [comment]
#172

File: dojox/1.11/rpc/Service.d.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+			 *     }
+			 * );
+			 * @param smd
+			 * @param options
+			 * @see https://dojotoolkit.org/reference-guide/1.10/dojox/rpc/Service.html#dojox-rpc-service
+			 */
+			constructor(smd: string | object, options: any);
+
+			// Internal method we are overriding
+			protected _generateService(serviceName: string, method: string): (method: string) => dojo.Deferred<any>;
```
Comment (msssk) [inline dojox/1.11/rpc/Service.d.ts:69]: Why is `method` typed as `string` in these methods? Looking in the source it appears to be an object with numerous properties.

**Review by msssk — COMMENTED** [review]
The use of `object` and `any` is less than optimal, but I suppose it's better to have some loose typings that allow you to use the module than no typings at all.

**Comment by edhager** [comment]
I'm closing the PR because I have created a new one that makes some additional changes: #182

**Pull request closed without merging**